export.metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+    "version": "7.0.2",
```

Comment:
Do we need metadata.json and 7.0.2 version in it?

Reply:
File Deleted and committed again